src/mono/System.Private.CoreLib/src/System/Exception.Mono.cs

```diff
@@ -5,12 +5,16 @@
 using System.Diagnostics;
 using System.Reflection;
 using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
 
 namespace System
 {
     [StructLayout(LayoutKind.Sequential)]
     public partial class Exception
     {
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal static extern uint GetExceptionCount();
+
```

Comment:
Might be better to have an enum or something to return the counters instead of a separate icall for each of them.

Reply:
This is how RuntimeEventSource does it and that code is shared with coreclr, meaning that they already have a dependency on these icalls. If we are going to change this wewill need to introduce RuntimeEventSource.Mono.cs and RuntimeEventSource.CoreClr.cs.

Reply:
Didn't go down that route, but maybe it is worth it to reduce number of icalls and it might be easier to expose more counters in the future as well.

Reply:
Those methods can be normal methods which call a shared icall ?

Reply:
yes, but we should split that source into a Mono and CoreClr shim that takes care of implementation differences between platforms if we don't want to have these as different icalls, so shared code will just call like GetExceptionCount and CoreClr shim implements it as an icall and we as a regular method that will call a shared icall with an enum returning requested counter.

Reply:
The icalls themselves are in mono specific files so they can be changed to a non-icall which calls an icall without the caller being affected.

Reply:
you are right, they are already in Mono specific files and coreclr already have their correspond files, and don't believe anything was actually in RuntimeEventSource, so that change should be straight forward, will go ahead and do it.

Reply:
Changed to implementation using one icall in EventPipeInternal using an enum to get different runtime counter values. So far I just added the once originally added by this PR, but going forward we could use this to surface other runtime counters into managed parts of EventPipe diagnostics.